backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+openapi: 3.0.0
+info:
+servers:
+paths:
+  /curation/auth/key:
+    post:
+      summary: Generate the curator's API key
+      security:
+        - cxguserCookie: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  key:
+                    type: string
+                    required: true
+                  id:
+                    type: string
+                    required: true
+        "401":
+          $ref: "#/components/responses/401"
+    get:
+      summary: Fetch the identifier of the curator's active API keys
+      security:
+        - cxguserCookie: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+        "401":
+          $ref: "#/components/responses/401"
+    delete:
+      summary: Delete the curator's API key
+      security:
+        - cxguserCookie: []
+      responses:
+        "202":
+          $ref: "#/components/responses/202"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/auth/token:
+    post:
+      summary: Generate an access token
+      security:
+        - apiKey: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  access_token:
+                    type: string
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections:
+    get:
+      description: Fetch all Collections metadata. Authentication required for private collections.
+      security:
+        - cxguserCookieLenient: []
+      parameters:
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  collections:
+                    type: array
+                      items:
```

Comment:
When I copy the raw file into [swagger.io](https://editor.swagger.io/), line 84 results in:
```
Parser error 
bad indentation of a mapping entry
```

Reply:
I'll proof this 👍